desktop-sync/latest/config/index.md

```diff
@@ -225,6 +225,22 @@ syncmanager.freeSpaceRestart=-3500M
 
 In this example, the `freeSpaceLimit` is set 3GB below the available space and the `freeSpaceRestart` is set 3.5GB below the current available space.
 
+## Force maximum size for user sync
+
+You can limit the amount of data that a user can sync by setting `syncmanager.maxSyncSize` to `10` bytes or more. When this is set and the data to sync exceeds the limit, you'll be notified in the content selection dialog, and the **Sync** button is disabled.
```

Comment:
The _maxSyncSize_ is checked only when the user selects what to sync in the content selection dialog, not all the time. I think this should be specified clearly.

Reply:
A better way would be to say "You can limit the amount of data that a user can select to sync by setting" or something similar (just a suggestion).